ndarray/src/main/java/org/tensorflow/ndarray/Shape.java

```diff
@@ -74,8 +77,8 @@ public static Shape scalar() {
    * Shape scalar = Shape.of()
    * }</pre>
    *
-   * @param dimensionSizes number of elements in each dimension of this shape, if any, or
-   *                       {@link Shape#UNKNOWN_SIZE} if unknown.
+   * @param dimensionSizes number of elements in each dimension of this shape, if any, or {@link Shape#UNKNOWN_SIZE} if
+   * unknown.
```

Comment:
This doesn't conform to Google's Java style. Can you fix it or back out all the formatting changes?